.github/pull_request_template.md

```
@@ -1,21 +1,55 @@
+- [ ] Database schema migration
+- [ ] Config changes (update `config.example.yaml`)
+- [ ] Proto regeneration (`buf generate`)
+- [ ] Infrastructure changes (Terraform / Helm)
+- [ ] Dependency updates that affect production builds
+
+## Related PRs
+<!-- Link any related PRs across repos (e.g., proto changes in candela-protos). Delete if not applicable. -->
+
+-
+
 ## Checklist
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)